python/cudf/cudf/core/index.py

```diff
@@ -293,6 +293,30 @@ def to_series(self):
 
         return Series(self._values)
 
+    def isnull(self):
+        """Identify missing values in an Index.
+        """
+        from cudf.core.series import Series
+
+        return Series(self.as_column().isnull(), name=self.name)
```

Comment:
Should this return a Series? Is that what Pandas does for calling `isna()` against an index?

Reply:
Looks like Pandas returns a numpy array here: https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.Index.isna.html

We should return an Index object instead.